core/src/main/java/smile/classification/DecisionTree.java

```diff
@@ -114,7 +113,7 @@
     /**
      * The root of the regression tree
      */
-    private Node root;
+    public Node root;
```

Comment:
no, no, no.

Reply:
This was needed for the application we use the library from as we need all of the information relating the tree. I can change it.